content/en/security/application_security/enabling/tracing_libraries/code_security/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Enabling ASM Code Security using Datadog Tracing Libraries
+kind: documentation
+type: multi-code-lang
+further_reading:
```

Comment:
Heads up: The index pages you have are missing the further reading partial at the bottom, so they aren't displaying.

Reply:
thank you!